### PR TITLE
App customization feature doc - copyedits, screenshot, use cases [2453890]

### PR DESCRIPTION
**Pull request by ashishguptaiitb**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c53852989aadecdbc5b6bb37f92cf0c8c2cb68e/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/db422e5c-c39f-481b-ad39-86b6511f006f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by ashishguptaiitb** [comment]
@surbhigupta12  and @v-habibaumme , kindly review.

File: msteams-platform/concepts/design/enable-app-customization.md [hunk 1]
```diff
@@ -9,28 +9,28 @@ ms.topic: overview
...
+* Changing the description to include language and guidance that is better understood by your organization's end-users.
+* Changing logo and logo outline to inspire confidence to the end-users to use the app since it seems provided by their organization.
 
-You can enable this feature by defining the app properties that your customers can customize in the [`configurableProperties` section in the Teams app manifest](/microsoftteams/platform/resources/schema/manifest-schema#configurableproperties), starting with version 1.11. That can be done in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home) if you've chosen to use the Developer Portal to edit the manifest of your app.
+You can enable this feature by defining the app properties in the [`configurableProperties` section in the app manifest file](/microsoftteams/platform/resources/schema/manifest-schema#configurableproperties). You can define the property using [Developer Portal for Teams](https://dev.teams.microsoft.com/home) if you've chosen to use the Developer Portal to edit the manifest of your app or or do it manually in the manifest file.
```
Comment (v-habibaumme) [inline msteams-platform/concepts/design/enable-app-customization.md:27]: "or" is a repeat in the last sentence

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ca674e2ae6bf0cf3a6bb8cc54a3953ccbf2497bf/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a176a483-9939-4706-95d5-e3ad489d48fd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/design/enable-app-customization.md (hunk 1 above)
Comment (ashishguptaiitb) [reply msteams-platform/concepts/design/enable-app-customization.md]: Thanks @v-habibaumme for the catch. I've updated the PR with more edits.

**Comment by ashishguptaiitb** [comment]
@surbhigupta12 , please review and merge.

File: msteams-platform/concepts/design/enable-app-customization.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
 
-### User considerations
-
-Provide guidelines for customers (specifically Teams admins) who want to customize your app. For more information, see [customize apps in Teams](/MicrosoftTeams/customize-apps).
-
 ## Block apps by default for users until an admin approves
 
-To enhance Teams app experience, you can hide an app from users by default until admin allows to unhide the app. For example, Contoso Electronics has created a help desk app for Teams. To enable appropriate functioning of the app, Contoso Electronics’ wants the customers to first set up specific properties of the app. The app is hidden by default and is available to users only after the admin allows it.
+To enhance Teams app experience, you can block an app from users until an admin approves the app. For example, Contoso Electronics has created a help desk app for Teams. To enable appropriate functioning of the app, Contoso Electronics’ wants the customers to first set up specific properties of the app. The app is blocked by default and is available to users only after the admin allows it. The functionality is called blocked by publisher.
```
Comment (surbhigupta12) [inline msteams-platform/concepts/design/enable-app-customization.md:36]: Is this information validated by our feature crew?

Can we please get an approval from them. Rest looks fine.

**Comment by surbhigupta12** [comment]
@ashishguptaiitb , can we please get an SME approval as well. We can then merge the PR.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f7880478076f6d971edb6d7e145b342c061ebaff/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 97 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ae5822fd-75c1-40b2-8c76-50b632d6f87f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fbdda071a7e6bfbe5874ab6330d2a3edb2e40c14/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83286d43-599d-45e8-ba75-2b10f6b204d2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/43b87e6f565fea1b9e199d822e213023600c01b2/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5686ccf-8e4c-43d4-9470-f95046e43779) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/26a970fd05fa01861078fe5dfcb114d7770abcb8/msteams-platform%2Fconcepts%2Fdesign%2Fenable-app-customization.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5b96fe2-87af-46ac-ae3b-e618d7ab32d7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**